src/movies.js

```diff
@@ -0,0 +1,144 @@
+import { renderElement } from './homepage.js';
+import { renderPopup, refresh } from './popup.js';
+
+const obArr = [
+  {
+    id: 0,
+    idTv: 1,
+    idInv: 'item_1',
+  },
+  {
+    id: 1,
+    idTv: 2,
+    idInv: 'item_2',
+  },
+  {
+    id: 2,
+    idTv: 3,
+    idInv: 'item_3',
+  },
+
+  {
+    id: 3,
+    idTv: 4,
+    idInv: 'item_4',
+  },
+  {
+    id: 4,
+    idTv: 5,
+    idInv: 'item_5',
+  },
+  {
+    id: 5,
+    idTv: 6,
+    idInv: 'item_6',
+  },
+];
+const Movies = {
+  init: () => {
+    Movies.showItemsCounter(obArr);
+    Movies.getShows(obArr);
+    Movies.getLikes();
+  },
+  showItemsCounter: (arr) => {
+    document.querySelector('#shows-counter').innerHTML = arr.length;
+    return arr.length;
+  },
+  getShows: (objArr) => {
+    const arr = Array.from(objArr);
+    arr.forEach((item) => {
+      fetch(`https://api.tvmaze.com/shows/${item.idTv}`)
+        .then((response) => response.json())
+        .then((json) => renderElement(json, item.idInv));
```

Comment:
- [ ] Your method is good but instead using just the fetch you need to make it `async and await and fetch` method, I did see  you are using down but also should be applied here.

![hhkkh](https://user-images.githubusercontent.com/61562626/130154229-e33ce9e5-cb46-4138-8453-b01757d71423.jpeg)